docs/tutorials/local_iteration_series/testing_with_artificial_conditions.md

```diff
@@ -0,0 +1,173 @@
+---
+id: testing_with_artificial_conditions
+title: Local Iteration - Testing multiplayer games with artificial network conditions
+description: Guide covering the available tools for local testing of multiplayer games with artificial latency, jitter and packet loss .
+---
+- [General advice](#general-advice)
+	- [Different network conditions for different peers](#different-network-conditions-for-different-peers)
+- [Unity Transport - Simulator Tools](#unity-transport---simulator-tools)
+	- [Clone-based workflow (ParrelSync)](#clone-based-workflow-parrelsync)
+	- [Debug Builds](#debug-builds)
+- [System-wide network conditioners](#system-wide-network-conditioners)
+	- [Clumsy on Windows](#clumsy-on-windows)
+		- [Settings quickstart](#settings-quickstart)
+	- [Network Link Conditioner on OSX](#network-link-conditioner-on-osx)
+		- [Settings quickstart](#settings-quickstart-1)
+
+
+When we are developing a multiplayer game that is intended to operate over the internet we inevitably have to deal with the reality of poor network conditions.
+
+Adverse network factors, such as [latency, jitter and packet loss](lagandpacketloss.md) all have to be taken into account during development and the tolerance thresholds for these should not be an afterthought or something relegated to an "optimization pass". 
+
+:::warning
+
+Not testing with artificial network conditions makes it significantly more likely to have unexpected behaviors when the game is running over the internet.
+
+:::
+
+This is challenging when we are [iterating on our game locally](local-Iteration-testing-multiplayer-games-locally.md) - after all, all of the game instances are still running on the same network interface. It's reasonable to expect that there will be little to no latency between the clients, which is no good for our purposes - we want the network to **misbehave**. 
+
+Thankfully there are a number of tools that can simulate adverse network conditions.
+
+For testing locally within the editor we suggest using [Unity Transport - Simulator Tools](#unity-transport---simulator-tools) along with [clone-based workflow via ParrelSync](testing_locally.md#parrelsync).
+
+For testing development builds with built-in artificial latency we suggest using [Unity Transport Simulator Tools with some custom code to inject artifical conditions into the build](#debug-builds).
+
+For testing release builds we suggest using either [Clumsy](#clumsy-on-windows) if you're on Windows, and [Network Link Conditioner](#network-link-conditioner-on-osx) if you're on Mac OS.
+
+## General advice
+
+It's not immediately obvious what the values and options we should enable in our network conditioning tools. All of them allow us to alter various aspects of network conditions such as latency, jitter and packet loss, though the names for concepts and the specific functionality varies between tools.
```

Comment:
nit: This general advice section could benefit from a bit of summarizing? @Briancoughlin @s-omeilia-unity what do you guys think?